### PR TITLE
Update web console config

### PR DESCRIPTION
**Pull request by spadgett**
Adopt API changes from https://github.com/openshift/api/pull/31

Requires https://github.com/openshift/origin/pull/18114
Requires https://github.com/openshift/openshift-ansible/pull/6730
Replaces #19 

/assign @deads2k 
@jwforres FYI

### CODE REVIEW

File: pkg/apis/webconsole/validation/validation.go [hunk 1]
```diff
@@ -50,14 +51,27 @@ func ValidateWebConsoleConfiguration(config *v1.WebConsoleConfiguration, fldPath
 		validationResults.AddWarnings(field.Invalid(fldPath.Child("metricsPublicURL"), "", "required to view cluster metrics in the console"))
 	}
 
-	for i, scriptURL := range config.ExtensionScripts {
-		if _, scriptURLErrs := ValidateSecureURL(scriptURL, fldPath.Child("extensionScripts").Index(i)); len(scriptURLErrs) > 0 {
+	if len(config.LogoutPublicURL) > 0 {
+		_, urlErrs := ValidateURL(config.LogoutPublicURL, fldPath.Child("logoutPublicURL"))
```
Comment (deads2k) [inline pkg/apis/webconsole/validation/validation.go:55]: why isn't this required to be secure?

**Comment by deads2k** [comment]
one question.  lgtm otherwise

File: pkg/apis/webconsole/validation/validation.go (hunk 1 above)
Comment (spadgett) [reply pkg/apis/webconsole/validation/validation.go:55]: Does it need to be? As I understand it, it is just a redirect URL after logout.

https://github.com/openshift/origin-web-console/blob/ea328a53a741c7c460914106fb0274f378195183/app/scripts/controllers/util/logout.js#L25-L26

FWIW, it wasn't required to be secure before.

https://github.com/openshift/origin-web-console-server/blob/master/pkg/apis/webconsole/validation/validation.go#L17

Comment (deads2k) [reply pkg/apis/webconsole/validation/validation.go:55]: > Does it need to be?

I like all things secure.  If it wasn't before, I guess it doesn't have to be today.

**Comment by spadgett** [comment]
origin and openshift-ansible changes have merged

/hold cancel

**Pull request merged**